tests/test_eject.py

```diff
@@ -0,0 +1,360 @@
+"""Tests for the eject command."""
+
+import os
+import shutil
+import subprocess
+from typing import Iterator
+from pathlib import Path
+from unittest.mock import call, patch
+
+import yaml
+import pytest
+from click.testing import CliRunner
+from aea.configurations.base import PublicId, _get_default_configuration_file_name_from_type  # noqa
+from aea.configurations.data_types import PackageType
+
+from auto_dev.commands.eject import cli
+
+
+@pytest.fixture
+def test_data_dir(tmp_path: Path) -> Iterator[Path]:
```

Comment:
theres already fixtures for this in conftest, please check them out, and add the defaults as constants iin the test data directory